.github/workflows/pr.yaml

```
@@ -1,147 +1,55 @@
+      - run: |
           docker network create frontend
-          docker compose run --user root --rm phpfpm composer install
-          docker compose run --user root --rm phpfpm composer normalize --dry-run
+          docker compose run --rm phpfpm composer validate composer.json --strict
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm composer normalize --dry-run
 
   validate-doctrine-schema:
     runs-on: ubuntu-latest
-    env:
-      DATABASE_URL: mysql://db:db@127.0.0.1:3306/db?serverVersion=10.11.0-mariadb
-    strategy:
-      fail-fast: false
-      matrix:
-        php: ["8.4"]
-    name: Validate Doctrine Schema (PHP ${{ matrix.php}})
-    services:
-      mariadb:
-        image: mariadb:10.11
-        env:
-          MYSQL_USER: db
-          MYSQL_PASSWORD: db
-          MYSQL_DATABASE: db
-          MYSQL_ROOT_PASSWORD: db
-        ports:
-          - 3306:3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+    name: Validate Doctrine Schema
     steps:
-      - name: Checkout
-        uses: actions/checkout@v4
-
-      - name: Setup PHP, with composer and extensions
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: ${{ matrix.php}}
-          extensions: apcu, ctype, iconv, imagick, json, redis, soap, xmlreader, zip
-          coverage: none
-
-      - name: Get composer cache directory
-        id: composer-cache
-        run: echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+      - uses: actions/checkout@v4
+      - run: |
+          docker network create frontend
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm bin/console doctrine:migrations:migrate --no-interaction
+          docker compose run --rm phpfpm bin/console doctrine:schema:validate
 
-      - name: Cache dependencies
-        uses: actions/cache@v3
-        with:
-          path: ${{ steps.composer-cache.outputs.dir }}
-          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
-          restore-keys: ${{ runner.os }}-composer
+  test-suite:
+    if: false # Currently the project has no tests
+    name: Test suite
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
 
-      - name: 'Composer install with exported .env variables'
+      - name: Test suite
         run: |
-          set -a && source .env && set +a
-          APP_ENV=prod composer install --no-dev -o
-
-      - name: Run Doctrine Migrations
-        run: APP_ENV=prod php bin/console doctrine:migrations:migrate --no-interaction
-
-      - name: Validate Doctrine schema
-        run: APP_ENV=prod php bin/console doctrine:schema:validate
+          docker network create frontend
+          docker compose run --rm phpfpm composer install
+          docker compose run --rm phpfpm ./vendor/bin/phpunit --coverage-clover=coverage/unit.xml
```

You don't use `./` when running `bin/console`, so for consistency

```suggestion
          docker compose run --rm phpfpm vendor/bin/phpunit --coverage-clover=coverage/unit.xml
```